telemetry_core/src/main/java/com/newrelic/telemetry/metrics/json/MetricBatchJsonTelemetryBlockWriter.java

```diff
@@ -38,7 +38,7 @@ public void appendTelemetryJson(MetricBatch batch, StringBuilder builder) {
             .collect(Collectors.joining(",")));
 
     if (retainedCount.get() != metrics.size()) {
-      logger.debug(
+      logger.info(
```

Comment:
my only concern with "you should fix this" is that if the "bad" data is coming from exporters, or from libraries (like micrometer), this will end up being 100% non-actionable by the end-user, and they'll just have to turn off all logging at INFO to get rid of the noise. Not saying it's wrong...just something to consider.

Reply:
Fair enough.  I think I'm erring on the side of making it visible by default so that newly developed code doesn't introduce a hidden problem.  I totally hear your concern tho.